CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [next]
  - Adding MouseMovementDetector
+ - Fixes #436 and improve Bgm code a bit
```

Comment:
I would go with a smaller and more formal sentence here like just: `Fixes BGM error`

Reply:
I wouldn't say that is "more formal" haha but I will replace